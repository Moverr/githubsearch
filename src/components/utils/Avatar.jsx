import React from 'react';
import { LazyLoadImage } from 'react-lazy-load-image-component';
 
const Avatar = ({ image }) => (
  <div>
    <LazyLoadImage   src={image}    />
    
  </div>
);
 
export default Avatar;