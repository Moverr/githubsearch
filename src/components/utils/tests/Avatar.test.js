import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
 
import Avatar from '../Avatar';
 


describe('Testing Avatar ', () => {
  it('should  render correctly', () => {    
    const props = { };

      const wrapper = shallow(<Avatar props />);
      expect(wrapper).toMatchSnapshot();
  });
})



 