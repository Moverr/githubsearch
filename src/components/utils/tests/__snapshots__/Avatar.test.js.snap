// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Avatar  should  render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing Input FIeld should  render correctly 1`] = `ShallowWrapper {}`;
